src/Test/Utilities/Desktop/InstrumentationChecker.cs

```diff
@@ -0,0 +1,267 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Xml.Linq;
+using Microsoft.CodeAnalysis.Emit;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities
+{
+    internal sealed class CSharpInstrumentationChecker : BaseInstrumentationChecker
```

Comment:
Could you add some doc comments, to these methods. To help those that are external or new to the project use them.